### ISSUE TITLE
Solution references projects not in repository

### ISSUE THREAD

**Issue by n00bje** [realworld666/MMSaveEditor#1]
The solution references the projects JsonCompare and NBug, which are both not in this repository.

**Comment by realworld666** [comment]
This is true. They can probably be removed

**Comment by TheTM1Channel** [comment]
I doubt that there will be much more work done on this since the game has essentially been abandoned by the devs, but I'll add my $0.02.

I noticed the same problem as n00bje. I therefore deleted the other two projects from the solution, saved and reloaded.

When I did a build the first problem that I had was 

"Error Condition="Exists('packages\Microsoft.Bcl.Build.1.0.14\tools\Microsoft.Bcl.Build.targets')" Text="The build restored NuGet packages. Build the project again to include these packages in the build. For more information, see http://go.microsoft.com/fwlink/?LinkID=317568." HelpKeyword="BCLBUILD2002" " 

Being a Microsoft error message the proposed "solution" was a barefaced lie, and the link provided led to "MSDN and TechNet blog sites have been retired, and blog content has been migrated and archived here." Good stuff, Microsoft. Further digging into the error code BCLBUILD2002 connected it with Microsoft.Bcl and Microsoft.Bcl.Build. Removing those references from packages.config and reloading the project cured **_that_** problem.

The next problem is that references to NBug are scattered throughout the project. Commenting them out was at least enough to get the application to BUILD (and run in debug mode), but whether I've broken anything in doing so is another matter. Time will tell.

If I find that there is anything that I can add to this I'll create some kind of "artificial" NBug class that will at least LOOK like it has the same functionality as the missing project, though to be honest this looks so comprehensive that I'm not sure I can add much of value.

This post may help if anyone else is looking at it as well.
